api/all.yaml

```diff
@@ -2082,6 +2085,75 @@ components:
       required:
         - full_name
         - version
+    CreateUser:
```

Comment:
shouldn't these definitions be in `control.yaml`?

Reply:
The reason I put them in control.yaml is to keep compiling the server module relatively simple. Right now only the Client module uses both all.yaml and control.yaml. If we want to break CreateUser out of all.yaml, we would need to add a separte build task dependency for server since the generated api code can't be in the same directory.

Reply:
@xx-db is going to see if he can move this to `control.yaml`. This is kind of a special case because we've set up the repository similar to other entities, which pass in things like `CreateFoo`, even though for users the over-the-wire entity is a SCIM `UserResource`. Another option would be to get rid of these objects and just pass in the SCIM `UserResource` directly to the repository.

Reply:
@xx-db refactored this into `control.yaml` 👍